### PR TITLE
Added Manage 2020 & 2019 Versions

### PR DESCRIPTION
**Pull request by Coolicky**
Added 2020 & 2019 Manage Versions. 

Tested on 2020 cant test on 2019 don't have access to that version but I don't see any issue with it not working.

### CODE REVIEW

**Comment by Coolicky** [comment]
Added Wix Setup Project Project & MSI multi version Installer

**Comment by Coolicky** [comment]
Added the same Icon instead of generic Navisworks one.

File: NwLookup_Setup/NwLookup_Setup.wixproj [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" InitialTargets="EnsureWixToolsetInstalled" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
+    <ProductVersion>3.10</ProductVersion>
+    <ProjectGuid>0cea8009-a942-43b4-8e1d-22610289f902</ProjectGuid>
+    <SchemaVersion>2.0</SchemaVersion>
+    <OutputName>RevitSetupProject</OutputName>
```
Comment (awmcc90) [inline NwLookup_Setup/NwLookup_Setup.wixproj]: Could you change this to NwLookup instead of RevitSetupProject

**Comment by Coolicky** [comment]
Changed RevitSetupProject to NwLookup. Updated Installer Included

**Pull request merged**